gapis/api/gles/api/egl.api

```diff
@@ -187,7 +187,12 @@ cmd EGLSurface eglCreateWindowSurface(EGLDisplay          display,
 
 @no_replay
 cmd EGLBoolean eglDestroyContext(EGLDisplay display, EGLContext context) {
-  return ? // TODO
+  if (context == null) {
```

Comment:
`if (context != null) && context in EGLContexts {` ?

Reply:
Agree, this is a bit weird.

Reply:
Ooops, sorry about that. Will send a fix.